AllReadyApp/Web-App/AllReady/gulpfile.js

```diff
@@ -49,13 +58,81 @@ gulp.task("watch", function () {
 })
 
 
-//https://www.npmjs.com/package/gulp-typescript
-//http://weblogs.asp.net/dwahlin/creating-a-typescript-workflow-with-gulp
-var tsProject = ts.createProject('tsconfig.json');
 
+//https://www.npmjs.com/package/gulp-typescript
 gulp.task('typescript:Build', function () {
-    var tsResult = tsProject.src() // instead of gulp.src(...) 
-        .pipe(ts(tsProject));
-    return tsResult.js.pipe(gulp.dest('wwwroot/release'));
+
+    var allTypeScript = 'ts/**/*.ts';
```

Comment:
To me all these definitions should be sucked up and put into a config object. 

`
paths.typescript = {
    source: 'ts/**/*.ts',
    definitions: 'tools/typings/**/*.ts'
}
`